scipy/stats/tests/test_distributions.py

```diff
@@ -131,6 +131,20 @@ def test_vonmises_pdf(x, kappa, expected_pdf):
     pdf = stats.vonmises.pdf(x, kappa)
     assert_allclose(pdf, expected_pdf, rtol=1e-15)
 
+# Expected values of the vonmises LOGPDF were computed
+# using wolfram alpha:
+# kappa * cos(x) - log(2*pi*I0(kappa))
+@pytest.mark.parametrize('x, kappa, expected_logpdf',
+                         [(0.1, 0.01, -1.8279520246003170),
+                          (0.1, 25.0, 0.5604990605420549),
+                          (0.1, 800, -1.5734679473375139),
```

Comment:
```suggestion
                          (0.1, 800, -1.5734567947337514),
```
This is the output of [`800* cos(1/10) - log(2*pi*I0(800))`](https://www.wolframalpha.com/input?i=800*+cos%281%2F10%29+-+log%282*pi*I0%28800%29%29).